google-login-plugin/resources/messages/UsageTrackerBundle.properties

```diff
@@ -0,0 +1,5 @@
+settings.enable.tracker.text=Would you like to help Google improve the Cloud Tools for IntelliJ plug-in by sharing anonymous usage data?
+settings.privacy.policy.comment=<html>Your use of this plugin is subject to the Apache 2.0 software license and the <a href=\{0}"">Google Privacy Policy</a>.</html>
+notification.group.display.id=Cloud Tools Plugin Usage Statistics
+notification.popup.title =Help improve the Cloud Tools plugin by sending anonymous usage statistics to Google
+notification.popup.content=<html>Please click <a href='allow'>I agree</a> if you want to help make the Cloud Tools plugin better or <a href='decline'>I don't agree</a> otherwise. Your use of this plugin is subject to the Apache 2.0 software license and the <a href='policy'>Google Privacy Policy</a>. <a href='settings'>Settings...</a></html>";
```

Comment:
1. Delete "Your use of this plugin is subject to the Apache 2.0 software license and the <a href='policy'>Google Privacy Policy</a>"
2. Can we add some line breaks? Will this work:

<html><p>Send anonymous usage data to Google:</p>
<a href='allow'>Agree</a><br />
<a href='decline'>Don't agree</a><br />
</html>